.circleci/docker_test_builds.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+################################################################################
+## Runs builds only if a specific file/directory has been updated in the latest
+## commit.
+##
+## Usage
+## -----
+## do_build package-name path/to/file/to/check
+################################################################################
+
+echo "Testing builds..."
+
+DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )/.." >/dev/null 2>&1 && pwd)"
+REPO_COMMIT=$(git rev-parse master)
+
+if [[ $ENVKEY = "" ]]; then
```

Comment:
Why is ENVKEY necessary ?
And how does it get set ?

Reply:
ENVKEY is necessary for at least some builds(e.g. dapp).  That's what that project I asked you to create the other day was about.  It has no data.  The env var is set in CircleCI.  Should be relatively secure, but I didn't want to take any chances.

Reply:
makes sense. ty for the details !